otherlibs/threads/pervasives.ml

```diff
@@ -117,14 +117,22 @@ external ( /. ) : float -> float -> float = "%divfloat"
 external ( ** ) : float -> float -> float = "caml_power_float" "pow"
   [@@unboxed] [@@noalloc]
 external exp : float -> float = "caml_exp_float" "exp" [@@unboxed] [@@noalloc]
+external exp2 : float -> float = "caml_exp2_float" "caml_exp2"
```

Comment:
Shouldn't all these functions be in the `Float` module?

Reply:
Yes, of course.  This pull request is really old, older than the `Float` module.